src/create-template/pvs.ts

```diff
@@ -78,8 +91,31 @@ export const processPvs = (
   verbose: boolean,
   root?: string
 ): ExtendedTemplateItem[] => {
+  const partIdToIterationId: { [k: string]: string } = {};
   const items: ExtendedTemplateItem[] = [];
 
+  const populatePartIdToIterationIds = (sps: SectionProperty[]): void =>
+    sps.forEach((sp) =>
+      sp.property_component_ref.forEach((pcr) => {
+        if (Array.isArray(pcr.property)) {
+          const pId = pcr.property.find(
+            (p) => p.name === 'part_displayIdentity'
+          );
+          const iId = pcr.property.find(
+            (p) => p.name === 'part_iterationDisplayIdentifierSansView'
+          );
+          if (pId && iId) partIdToIterationId[pId.value] = iId.value;
+        }
+      })
+    );
+
+  const getRevisionId = (name: string): string => {
+    for (const k of Object.keys(partIdToIterationId)) {
+      if (k.includes(name)) return partIdToIterationId[k];
+    }
+    return DefaultPartRevision;
```

Comment:
This works the same if these two properties don't exist so is safe to publish, but will want to wait to ensure these are the correct ones.